admin-console/src/main/webapp/package.json

```
@@ -23,6 +23,7 @@
     "react-scripts": "4.0.3",
     "react-use": "17.3.1",
     "typescript": "4.4.3",
+    "typestyle": "^2.1.0",
```

Lets pin the dependency. (will need a `yarn install` to refresh the `yarn.lock`) 

```suggestion
    "typestyle": "2.1.0",
```